rali/rali/source/rali_api_augmentation.cpp

```diff
@@ -412,7 +428,11 @@ raliCropResizeFixed(
         // user provides the output size and the dimension of all the images after this node will be fixed and equal to that size
         output->reset_image_roi();
 
-        context->master_graph->add_node<CropResizeNode>({input}, {output})->init(area, aspect_ratio, x_center_drift, y_center_drift);
+        std::shared_ptr<CropResizeNode> crop_resize_node =  context->master_graph->add_node<CropResizeNode>({input}, {output});
+        crop_resize_node->init(area, aspect_ratio, x_center_drift, y_center_drift);
+        if (context->master_graph->meta_data_graph())
```

Comment:
Codacy found an issue: [Either the condition '!context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5976200)